rotkehlchen/types.py

```diff
@@ -199,6 +200,8 @@ def deserialize_evm_tx_hash(val: Union[Web3HexBytes, bytearray, bytes, str]) ->
 T_TradeID = str
 TradeID = NewType('TradeID', T_TradeID)
 
+ZERO_PRICE = Price(ZERO)
```

Comment:
Why did you move this here? This is a constant, not a type. It should be inside the constants where it was before.